tests/config/operator_install/imagedigestmirrorset.yaml

```diff
@@ -0,0 +1,23 @@
+# The images are pulled from our redhat-user-workloads registry or staging registry while the production image is not yet available.
+kind: ImageDigestMirrorSet
+apiVersion: config.openshift.io/v1
+metadata:
+  name: openshift-lightspeed-prod-to-ci
+spec:
+  imageDigestMirrors:
+    - source: registry.redhat.io/openshift-lightspeed-beta/lightspeed-rhel9-operator
+      mirrors:
+        - registry.staging.redhat.io/openshift-lightspeed-beta/lightspeed-rhel9-operator
```

Comment:
Remove the above line - same goes to all the ones `- registry.staging.redhat.io` below.